docs/config.md

```diff
@@ -133,30 +133,34 @@ If specified, a field's value must be set to be a hex representation of a RGB co
 
 ### Component Styles
 
-To define application's component styles, user needs to specify **all of the below fields**:
+To define application's component styles, the user can specify any of the below fields:
 
 - `block_title`
+- `border`
 - `playback_track`
 - `playback_album`
 - `playback_metadata`
 - `playback_progress_bar`
 - `current_playing`
 - `page_desc`
 - `table_header`
+- `selection`
 
 A field in the component styles is a `Style` struct which has three optional fields: `fg`, `bg` and `modifiers`. `fg` and `bg` can be either a palette's color (string in pascal case) or a custom RGB color using the following format: `fg = { Rgb { r = ..., g = ..., b = ... } }`. `modifiers` can only be either `Italic` or `Bold`.
 
 Default value for application's component styles:
 
 ```toml
 block_title = { fg = "Magenta"  }
+border = {} # uses the palette's fg and bg color
```

Comment:
Instead of putting a comment here, can you add a note after component style's document that mentions that the default values for `fg` and `bg` are palette's `fg` and `bg`?

Reply:
implemented in https://github.com/aome510/spotify-player/pull/118/commits/591568db595cbe27b517b96e07f6837008b97272 - not sure about the location of the comment, hope, that's OK